Packs/BitSight/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,27 @@
+
+#### Incident Fields
```

Comment:
I think this is a breaking change. @amshamah419 Right?

Reply:
I would think so. Looks like the mapping has changed quite a bit (even if it's just adding BitSight to the field ID)